pkg/mount/scuzz_mount_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build amd64
+
+package mount_test
+
+import (
+	"testing"
+
+	"github.com/u-root/u-root/pkg/mount/block"
+	"github.com/u-root/u-root/pkg/mount/scuzz"
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestIdentify(t *testing.T) {
+	testutil.SkipIfNotRoot(t)
```

Comment:
scuzz_mount_test.go is missing the accompanying _integration_test.go (copy from mount_integration_test.go; use only -hda since only /dev/sda is used here)

Reply:
For amd64 archs the current mount integration test still runs TestIdentify since they're still in the same in mount_test package (just tested). Do you still want a separate integration test file for the scuzz tests?

Reply:
Oooh. You're right, sorry, I didn't pay attention to that.